### PR TITLE
Fix Dockerfile to catch up to 6.0.0

### PR DESCRIPTION
**Pull request by hobu**
This should fix `osgeo/proj` docker container auto-building

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,40 +1,54 @@
...
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        libsqlite3-0 \
+        curl unzip
+
+COPY --from=builder  /build/usr/bin/ /usr/bin/
+COPY --from=builder  /build/usr/lib/ /usr/lib/
+COPY --from=builder  /build/usr/include/ /usr/include/
+COPY --from=builder  /build/usr/share/proj/ /usr/share/proj/
+
+RUN \
```
Comment (rouault) [inline Dockerfile:42]: you should use the proj-datumgrid-XXXXXX packages instead. Some of the below grids are deprecated

Comment (hobu) [reply Dockerfile:42]: will do.

File: Dockerfile [hunk 2]
```diff
@@ -40,15 +40,12 @@ COPY --from=builder  /build/usr/include/ /usr/include/
...
-    curl -LOs http://download.osgeo.org/proj/vdatum/vertcon/vertconc.gtx && mv vertconc.gtx /usr/share/proj; \
-    curl -LOs http://download.osgeo.org/proj/vdatum/vertcon/vertcone.gtx && mv vertcone.gtx /usr/share/proj; \
-    curl -LOs http://download.osgeo.org/proj/vdatum/vertcon/vertconw.gtx && mv vertconw.gtx /usr/share/proj; \
-    curl -LOs http://download.osgeo.org/proj/vdatum/egm96_15/egm96_15.gtx && mv egm96_15.gtx /usr/share/proj; \
-    curl -LOs http://download.osgeo.org/proj/vdatum/egm08_25/egm08_25.gtx && mv egm08_25.gtx /usr/share/proj
+    curl -LOs http://download.osgeo.org/proj/proj-datumgrid-1.8.zip &&  unzip -j -u -o proj-datumgrid-1.8.zip  -d /usr/share/proj; \
+    curl -LOs http://download.osgeo.org/proj/proj-datumgrid-europe-1.2.zip &&  unzip -j -u -o proj-datumgrid-europe-1.2.zip -d /usr/share/proj; \
+    curl -LOs http://download.osgeo.org/proj/proj-datumgrid-oceania-1.0.zip &&  unzip -j -u -o proj-datumgrid-oceania-1.0.zip -d /usr/share/proj; \
+    curl -LOs http://download.osgeo.org/proj/proj-datumgrid-world-1.0.zip &&  unzip -j -u -o proj-datumgrid-world-1.0.zip -d /usr/share/proj;
```
Comment (rouault) [inline Dockerfile]: http://download.osgeo.org/proj/proj-datumgrid-north-america-1.2.zip as well

**Comment by kbevers** [comment]
Thanks for the update, Howard.

**Pull request merged**